docs/user-docs/test_matrix.md

```diff
@@ -36,7 +36,7 @@ Devices and configurations on which TrenchBoot is known to not work:
 
 [aem]: https://github.com/TrenchBoot/qubes-antievilmaid
 [qubesos]: https://www.qubes-os.org/
-[openqa]: https://open.qa/
+[openqa]: ./openqa.md
```

Comment:
The link should also be included in `mkdocs.yml`, without it there is no navigation menu on the left.